### DISCUSSION TITLE
Unable to register as student due to location setting problem

### DISCUSSION THREAD

**Discussion by Spotlight-Kid** [community/community#50578] [GitHub Education] [answered]
I can't get past the "Get your GitHub benefits" page. I keep getting a "To continue you must modify your browser settings to allow GitHub to use your location information." message.
I am on macOS Ventura 13.2.1 using Firefox 111.0 and allowed the page to access location when asked. I also select "Remember this".
I also tried Safari and in settings set the default to allow access to location settings, and still no joy.

**Comment by Hanzy1996** [comment]
I have the same problem

**Reply by Spotlight-Kid** [reply, marked as answer]
I've managed to fix the issue by going into System settings->Privacy & Security->Location Services and giving Firefox access

**Reply by lvucr** [reply]
Same problem, I tried going into "System Settings ->Privacy & Security->Location Services" but at this point, I do have not the option to give access to Chrome.

**Comment by Shiranavni** [comment]
I have the same problem on safari and though I change the settings I still get that error

**Comment by cborot** [comment]
Idem for me on firefox. This settings has no effect.

**Reply by mplatab** [reply]
System settings->Privacy & Security->Location  esto tiene que hacer en la configuración del sistema como en la configuración de Windows si tienen Windows y lo hacen con el navegador que tienen como predeterminado

**Comment by Servinjesus1** [comment]
It's caused by VPNs. If you get this error message and you have System Settings already enabled to share location services, disable your VPN.

**Comment by kairoslav** [comment]
Works on mobile!

**Reply by siamsledvk** [reply]
mine still doesn't

**Comment by SyedMohsinKamal2005** [comment]
In my case using incognito done the job.

**Reply by siamsledvk** [reply]
I tried it now and it worked.
